Segment-ComScore/Classes/SEGComScoreIntegration.h

```diff
@@ -11,11 +11,24 @@
 #import <ComScore/ComScore.h>
 
 
+@protocol SEGStreamingAnalyticsFactory <NSObject>
+- (SCORStreamingAnalytics *)create;
+@end
+
+
+@interface SEGRealStreamingAnalyticsFactory : NSObject <SEGStreamingAnalyticsFactory>
+@end
+
+
 @interface SEGComScoreIntegration : NSObject <SEGIntegration>
 
 @property (nonatomic, strong) NSDictionary *settings;
 @property (nonatomic, strong) Class scorAnalyticsClass;
+@property (nonatomic, strong) SCORStreamingAnalytics *streamAnalytics;
+@property (nonatomic) id<SEGStreamingAnalyticsFactory> streamingAnalyticsFactory;
+NSNumber *convertFromKBPSToBPS(NSDictionary *src, NSString *key);
```

Comment:
@ladanazita what's this line doing here? It does not appear to be used anywhere else